sklearn/linear_model/tests/test_ridge.py

```diff
@@ -692,6 +697,48 @@ def test_sparse_design_with_sample_weights():
                                       decimal=6)
 
 
+def test_ridgecv_alpha_conversion_to_array():
```

Comment:
I think you can remove this line and dedent the rest of this function

Reply:
Great, I'm new to tests in python. Already fixed it.